src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px;
```

Comment:
The `border-radius` property seems unnecessary here. The star images you are using do not require any border rounding. Please, remove this line if it doesn't affect your layout.